packages/account/src/Configs/poa-common-mistake-examples-config.tsx

```diff
@@ -19,19 +21,19 @@ type TExampleImageConfig = {
  */
 export const getExampleImagesConfig = (is_eu: boolean): Array<TExampleImageConfig> => [
     {
-        image: IcDocumentNameMismatch,
+        image: DerivLightIcDocumentNameMismatchIcon,
```

Comment:
Hi @likhith-deriv 👋 ,

Just curious about size. Without putting size or width and height. 
is that working fine ? For quill icons we usually add size.

But maybe your case it works fine. 😁

Reply:
The images are spanning the size of the grid to which they are allocated

Reply:
Alright man, 
Thank you for your response 🙇